.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ env:
   # maven.wagon.rto is in millis, defaults to 30m
   MAVEN_OPTS: "-Xmx512M -XX:+ExitOnOutOfMemoryError -Dmaven.wagon.rto=60000"
   MAVEN_INSTALL_OPTS: "-Xmx2G -XX:+ExitOnOutOfMemoryError -Dmaven.wagon.rto=60000"
-  MAVEN_FAST_INSTALL: "-B --strict-checksums -V --quiet -T C1 -DskipTests -Dair.check.skip-all"
+  MAVEN_FAST_INSTALL: "-B --strict-checksums -V --quiet -T C1 -Dquick -Dmaven.test.skip=false"
```

Comment:
document why `maven.test.skip=false`, it's not intuitive

per https://github.com/trinodb/trino/pull/10519#discussion_r781284002, it seems necessary to retrigger downloads

also i just realized that, since `-Dquick` skips whole plugins altogether, it's will actually skip downloading their dependencies. This means the build will be faster, but subequent `mvn test` may fail due to dep download.

Reply:
I reverted this change.